mungegithub/submit-queue/ingress.yaml

```diff
@@ -28,3 +31,10 @@ spec:
         - backend:
             serviceName: test-infra-sq-status
             servicePort: 80
```

Comment:
There should be an entry here for host: submit-queue.k8s.io

It looks like it's currently not behind a LoadBalancer Ingress, so the DNS will have to be updated as well.